app/uk/gov/hmrc/agentsubscriptionfrontend/views/helpers/input.scala.html

```diff
@@ -0,0 +1,100 @@
+@*
```

Comment:
This seems to be a fork of input.scala.html from play-ui. It would be nice to use the play-ui version instead.
If that's not possible we should make sure we don't end up with a copy of it in each of our frontends because that would be hard to maintain - we can put it in a different library.